static-site/src/pages/team.jsx

```diff
@@ -20,16 +20,27 @@ const TeamPage = () => {
   console.log("<TeamPage>");
   return (
     <div>
-      <H1>Nextstrain core team</H1>
+      <H1>Project Leads</H1>
 
       <FlexCenter>
         <CenteredFocusParagraph>
-          {"Nextstrain was co-founded by:"}
+          {"The Project Leads are the two co-founders of Nextstrain:"}
         </CenteredFocusParagraph>
       </FlexCenter>
 
       <ListOfPeople people="founders" teamPage />
 
+      <H1>Scientific Advisory Board</H1>
+      <FlexCenter>
+        <CenteredFocusParagraph>
+          {`In March 2024, a scientific advisory board was founded to provide guidance on future directions.`}
```

Comment:
```suggestion
          {`In March 2024, a scientific advisory board was founded to provide guidance on future directions for the Nextstrain project.`}
```

Reply:
Resolved in 867effb5597b905fb45aba4fa1696b85ca3d5633. Also fixed date of board formation (founded Oct 2023, first meeting was Mar 2024).